crates/services/p2p/src/service.rs

```diff
@@ -439,6 +474,12 @@ where
             RequestMessage::SealedHeaders(range) => {
                 self.handle_sealed_headers_request(range, request_id)
             }
+            RequestMessage::AllTransactionsIds => {
+                self.handle_all_transactions_ids_request(request_id)
+            }
+            RequestMessage::FullTransactions(tx_ids) => {
```

Comment:
I think we should prefix the `RequestMessage`'s with `TxPool` to disambiguate between database transactions and txpool transactions :)

Reply:
Done